src/components/Calculator.js

```diff
@@ -0,0 +1,49 @@
+import React from 'react';
+import './Calculator.css';
+import PropTypes from 'prop-types';
+
```

Comment:
- [ ] You have not implemented the events. You need to import the provided `calculate.js` file and use it to implement your calculation events.